config/default/manager_auth_proxy_patch.yaml

```diff
@@ -16,7 +16,7 @@ spec:
             capabilities:
               drop:
                 - "ALL"
-          image: gcr.io/kubebuilder/kube-rbac-proxy:v0.13.1
+          image: gcr.io/kubebuilder/kube-rbac-proxy:v0.14.1
```

Comment:
The latest is v0.14.2, but I would avoid jumping to the latest and greatest, unless we absolutely have to. Currently trying to understand if there are any reported security vulnerabilities for v0.14.2

https://artifacthub.io/packages/helm/nutanix/ndb-operator?modal=security-report